### DISCUSSION TITLE
Regarding Observed data adding in

### DISCUSSION THREAD

**Discussion by TitusRoy** [Open-Systems-Pharmacology/Forum#1118] [Questions & Problems] [answered]
Hello all,

Using QSP 10,

During my building blocks, when I insert observed data set ( a simple excel sheet ), it is not accepting. Says " not compatible "

Kindly help me with this.

Thanks.

**Comment by msevestre** [comment, marked as answer]
Hi @TitusRoy 

Please refer first to the extensive OSPSuite documentation, explaining in details how to import observed data
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/import-edit-observed-data

If you have any issue, it is important that you describe exactly what you are doing so that we can reproduce. From your description, we do not know what tool you are using, what data you are trying to import, where is the message showing up etc.... 

Have fun :)

**Comment by TitusRoy** [comment]
Dear Micheal,

Thank you for your email and the link.


Finally I made it work. The error was in my excel file, where I had 'full
stop' instead 'comma'.

Thanks for your instant help.


Can you please let me know why the decimal explanatory comes in comma
instead full stop? In real settings, comma gives it greater another value.



Appreciate your help.


With Best regards,
Roy.

**Reply by msevestre** [reply]
Hi @TitusRoy 
Can you upload your file here so that we can check it out? Full stop as decimal separator is definitely supported so your file might have other issues
